lucene/core/src/java/org/apache/lucene/store/FSDirectory.java

```diff
@@ -414,12 +430,12 @@ private void privateDeleteFile(String name, boolean isPendingDelete) throws IOEx
      * a native buffer outside of stack if the write buffer size is larger.
      */
     static final int CHUNK_SIZE = 8192;
-    
-    public FSIndexOutput(String name, boolean useEncryption) throws IOException {
-      this(name, useEncryption, StandardOpenOption.WRITE, StandardOpenOption.CREATE_NEW);
+      
```

Comment:
Unnecessary whitespaces. Also in ` lucene/core/src/java/org/apache/lucene/util/crypto/Crypto.java `.